light-zk.js/src/utxo.ts

```diff
@@ -497,6 +526,7 @@ export class Utxo {
         ]),
       );
       this._commitment = commitment;
+      // @matteo: this is going to return the same variable in either case
```

Comment:
yes we are saving this so that we don't need to recompute the hash in case we call it multiple times

Reply:
old stuff, removed ✅